core_backend/app/config.py

```diff
@@ -120,3 +120,9 @@
 
 # Language
 DEFAULT_DOC_LANGUAGE = os.environ.get("DEFAULT_DOC_LANGUAGE", "ENGLISH")
+
+# HTTPX
+HTTPX_TIMEOUT: float = 10.0
+
+# Turn.io
+TURNIO_API_BASE: str = os.environ.get("TURNIO_API_BASE", "https://whatsapp.turn.io/v1/")
```

Comment:
For AAQ, we only need `TURNIO_API_BASE`, which is fixed. The `TURNIO_API_KEY` will be provided in each `QueryBase` object instead.